dbt/adapters/glue/impl.py

```diff
@@ -154,25 +122,11 @@ def get_relation(self, database, schema, identifier):
             logger.error(e)
             logger.error(f"relation {schema}.{identifier} not found")
 
-    @available
-    def drop_view(self, relation: BaseRelation):
-        session, client, cursor = self.get_connection()
-        code = f'''DROP VIEW IF EXISTS {relation.schema}.{relation.name}'''
-        try:
-            cursor.execute(code)
-        except Exception as e:
-            logger.error(e)
-            logger.error("drop_view exception")
-            logger.error(f"""drop_view exception
-                             relation schema : {relation.schema}
-                             relation identfier : {relation.name}
-            """)
-
     @available
     def drop_relation(self, relations):
         session, client, cursor = self.get_connection()
         for relation in relations:
-            code = f'''DROP TABLE IF EXISTS {relation}'''
```

Comment:
I'm not sure why there was both a `drop_view` and a `drop_relation`. I'm also not sure why a connection is created inside of this method (and all methods in the `GlueAdapter` class). In [SQLAdapter, the convention](https://github.com/dbt-labs/dbt-core/blob/1390715590adcea140e4e694a20b35cef8bdbfbc/core/dbt/adapters/sql/impl.py#L133-L140) is that the method just calls `execute_macro` which calls the dbtjinja equivalent.

My feeling is that you could get rid of almost all of these methods if you could ensure that `execute_macro` `GlueConnectionManager.open()` and ``GlueConnectionManager.execute()` are working properly